setup.py

```diff
@@ -42,6 +42,7 @@
               'snapcraft.internal',
               'snapcraft.internal.cache',
               'snapcraft.internal.deltas',
+              'snapcraft.internal.new_sources',
```

Comment:
:+1: , I didn't realize GitHub would follow renames without dumping a bunch of red and green. This is a much better path than the intermediate package.